portal/public/index.html

```diff
@@ -145,20 +149,45 @@
     </style>
 </head>
 <body>
+    <noscript>
+        <strong>Het zoekportaal heeft JavaScript nodig om te functioneren. Zet JavaScript aan in de browser.</strong>
+    </noscript>
+
+    <div id="unsupported">
+        <img src="/images/surflogo@3x.png">
+        <p>Internet Explorer en Legacy Edge worden niet ondersteund door het zoekportaal. Gebruik een meer recente browser.</p>
+        <p>Internet Explorer and Legacy Edge are not supported by the search portal. Please use a more recent browser.</p>
+    </div>
 
     <div class="mobile">
         <div class="mobile-content">
-          <img src="/images/surflogo@3x.png">
-          <p>Het zoekportaal is gedurende de pilot niet te gebruiken op kleine schermen. Probeer je device te draaien of gebruik een computer.</p>
-          <p>During the pilot the search portal can't be used on small screens. Try to rotate your device or use a desktop computer.</p>
-        </div>
-    </div>
-
-    <div id="__nuxt">
-        <div id="nuxt-loading" aria-live="polite" role="status">
-            <div>Loading...</div>
+            <img src="/images/surflogo@3x.png">
+            <p>Het zoekportaal is gedurende de pilot niet te gebruiken op kleine schermen. Probeer je device te draaien of gebruik een computer.</p>
+            <p>During the pilot the search portal can't be used on small screens. Try to rotate your device or use a desktop computer.</p>
         </div>
     </div>
 
+    <script>
+      var isIE = typeof document.documentMode == "number" || new Function("return/*@cc_on!@*/!1")( );
+      var isEdge = !isIE && !!window.StyleMedia;
+
+      if (isIE || isEdge ) {
+          document.getElementById("unsupported").style.display = "block";
+      }
+      else {
+          var app = document.createElement("div");
```

Comment:
Nu is deze HTML verdwenen 😕 . Kun je dit element niet laten staan en als het IE is dit element weghalen?